resources.go

```diff
@@ -128,18 +132,29 @@ func azureType(mod string, typ string) tokens.Type {
 	return tokens.Type(azureMember(mod, typ))
 }
 
+// azureSubType manufactures a type token for the Azure package and the given module, submodule, and type.
+func azureSubType(moduleTitle, sub, typ string) tokens.Type {
+	moduleName := strings.ToLower(moduleTitle)
+	namespaceMap[moduleName] = moduleTitle
+	return azureType(moduleName+"/"+sub, typ)
```

Comment:
How about refactoring s.t. `azure{Type,Member}` take the module name and type/function name separately? Then we could just do `strings.ToLower` in one/both of those functions and let it do the `module+"/"+thing` dance.

Reply:
Good call! I moved all the logic to `azureMember` and other functions are just one-liners with descriptive names.